workorders.lic

```diff
@@ -158,10 +171,17 @@ class WorkOrders
   end
 
   def find_recipe(info, item, quantity)
-    recipe = @recipes.find { |r| r['name'] == item }
-    items_per_stock = info['stock-volume'] / recipe['volume']
-    spare_stock = (info['stock-volume'] % recipe['volume']).nonzero?
-    scrap = spare_stock || (quantity % items_per_stock).nonzero?
+    if !@use_bone
```

Comment:
Is the only difference in these two cases `stock-volume` vs `stock-bone-volume`?